src/components/PizzaItem/PizzaItem.jsx

```diff
@@ -0,0 +1,41 @@
+import {useDispatch} from 'react-redux';
+import {useState} from 'react';
+
+
+function PizzaItem (props) {
+    const dispatch=useDispatch();
+    
+    const [pizzaState, setPizzaState] = useState(true)
+    const handlePizzaAdd = event => {
+        event.preventDefault();
+        dispatch({type:'ADD_TO_CART', payload: props.pizza });
+        console.log(props.pizza);
+        setPizzaState(!pizzaState);
+    }
+    const handlePizzaRemove = event => {
+        event.preventDefault();
+        dispatch({type:'REMOVE_PIZZA', payload: props.id});
+        setPizzaState(!pizzaState);
+    }
+    return(
+        pizzaState ? (
```

Comment:
Maybe it will be simpler if you use the conditional rendering on the button only